native/core/src/execution/datafusion/shuffle_writer.rs

```diff
@@ -798,7 +810,6 @@ impl ShuffleRepartitioner {
 
     /// Writes buffered shuffled record batches into Arrow IPC bytes.
     async fn shuffle_write(&mut self) -> Result<SendableRecordBatchStream> {
-        let _timer = self.metrics.baseline.elapsed_compute().timer();
         let num_output_partitions = self.num_output_partitions;
         let mut buffered_partitions = self.buffered_partitions.lock().await;
         let mut output_batches: Vec<Vec<u8>> = vec![vec![]; num_output_partitions];
```

Comment:
we were previously including the time to run these lines in the total time but now we are not?

Reply:
I checked how Spark calculates shuffle write time. Spark only includes the time spent on disk operations. I tried to follow it so skipped the lines that are not relevant to disk op.